package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha",
-    "start": "node server.js"
+    "start": "start http://localhost:3000 && node server.js"
```

Comment:
you dont need to start the browser. it is enough to start the node and pass the server.js to it